build.gradle

```diff
@@ -11,7 +11,7 @@ buildscript {
     }
     dependencies {
         classpath "com.android.tools.build:gradle:7.0.2"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.21"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10"
```

Comment:
I updated kotlin to 1.6.10 (latest) because I had this issue before:

https://youtrack.jetbrains.com/issue/KT-45545

Which I'm sure has nothing to do with my library / changes 🤷 
Anyway, it should be the same for you, but let me know if you find any issues. 
Also updated compose to match the new kotlin version.